platformcomponents/desktop/content-card-item.json

```diff
@@ -0,0 +1,20 @@
+{
+  "content-card-item": {
```

Comment:
content-file-card-item

Reply:
Pushed commit
`Renamed content-card-item to content-file-card-item`
Changing this conversation to Resolved.